pyproject.toml

```diff
@@ -1,2 +1,91 @@
 [tool.black]
 target-version = ['py310']
+
+# This pyproject.toml is setup so it can be used with or without poetry and also
+# supports editable installs (PEP 660) without breaking IDE and linter inspection.
+
+# DEVELOPER INSTALL INSTRUCTIONS (assuming current directory is at the top of the project) :
+# Poetry:
+#   poetry install --extras dev
+# PIP:
+#   pip install -e .[dev]
+
+[tool.poetry]
+name = "screenpy_selenium"
+version = "4.0.3"
+description = "ScreenPy extension to enable interacting with Selenium."
+authors = ["Perry Goy <perry.goy@gmail.com>"]
+maintainers = ["Gabe Langton", "Marcel Wilson"]
+license = "MIT"
+repository = "https://github.com/ScreenPyHQ/screenpy_selenium"
+documentation = "https://screenpy-selenium-docs.readthedocs.io"
+readme = "README.md"
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Quality Assurance",
+    "Topic :: Software Development :: Testing",
+    "Topic :: Software Development :: Testing :: BDD",
+    "Natural Language :: English",
+    "License :: OSI Approved :: MIT License",
+]
+
+# It's possible to add optional dependencies with the poetry CLI tool using:
+#   poetry add --optional some_dev_pkg
+# This will set `optional = true` flag in [tool.poetry.dependencies], as seen below
+# But then you need to remember to manually edit the [tool.poetry.extras] dev section 
+# and declare the package. This allows `pip install .[dev]` to work as expected
+# Poetry users will need to use the `--extras dev` option rather than the `--with dev`
+# so we dont have two different sets of package versions to update.
+
+[tool.poetry.dependencies]
+python = "^3.7"
+screenpy = ">=4.0.2"
+screenpy_pyotp = ">=4.0.0"
+selenium = ">=4.1.0,<4.5"
```

Comment:
I'm trying to recall, but is there something about selenium 4.5 that was not backwards compatible?  or was that a fever dream?

Reply:
Well, it isn't out yet, so that might be what you're thinking of. :p

Minor version updates shouldn't break compatibility, at least philosophically speaking. A major version update might. But the idea was that we wouldn't need to chase after Selenium's updates to keep ScreenPy Selenium working with the latest, and we could just update things if they ever broke.

Reply:
"Looks like I picked the wrong week to quit sniffing glue" -- Steven McCrosky (Airplane)